index.js

```diff
@@ -69,9 +71,11 @@ const registerBearerStrategy = function(startWith, server, options) {
             server.auth.strategy('bearer', 'bearer-access-token', {
                 validateFunc: (token, callback) => {
                     if (options.tokens.hasOwnProperty(token)) {
-                        return callback(null, true, {
-                            newsroom: options.tokens[token]
-                        });
+                        let credentials = {newsroom: options.tokens[token]};
+                        if (typeof options.credentials === 'object') {
```

Comment:
I am not sure if that check has to be here - 🐻  in mind that if `credentials` is not an object the code will silently not do anything (which is probably not what we want). So either a proper validation (supposedly somewhere on top of `registerBearerStrategy` function or no validation (and letting it crash).